Editor/Window/ConnectToFleetInput.cs

```diff
@@ -48,6 +48,13 @@ public ConnectToFleetInput(VisualElement container, StateManager stateManager)
             _stateManager.OnUserProfileUpdated += async () =>
             {
                 await UpdateFleetMenu();
+                
+                var fleet = _fleetAttributes.FirstOrDefault(fleet => fleet.Name == _stateManager.AnywhereFleetName);
+                if (fleet == null)
+                {
+                    _fleetState = FleetStatus.NotCreated;
+                }
```

Comment:
```suggestion
                }
                else
                {
                    _fleetState = FleetStatus.Selected;
                }
```
Discussed with Shashank, resetting the status on profile update since being in the middle of selecting/creating doesn't make sense when swapping profiles